functions/src/send-email-to-manager.ts

```diff
@@ -68,13 +68,18 @@ export const sendEmailToManager = functions.firestore
      */
     let emailSent = false;
     await firebase.firestore().runTransaction(async transaction => {
+      //Make sure we didn't send a mail already
+      if (vote.emailToManagerSent) {
```

Comment:
This will not work because `vote` was read from the database before the beginning of the transaction, which means it could be stale. The code has to read the vote again inside the transaction.